javascript/javascript1/week3/Songs.js

```diff
@@ -0,0 +1,64 @@
+const songDatabase = [
+  {
+    songId: 1,
+    title: "My baby",
+    artist: "Soggy socks"
+  },
+  {
+    songId: 2,
+    title: "3 nails in wood",
+    artist: "The carpenters"
+  },
+  {
+    songId: 3,
+    title: "Blacker than black",
+    artist: "Instant coffee"
+  },
+  {
+    songId: 4,
+    title: "When is enough too little?",
+    artist: "The spies girls"
+  }
+];
+
+//add song
+let updatedSongDatabase = addSongToDatabase(songDatabase);
+console.log("Songs database with a new song: ");
+console.log(updatedSongDatabase);
+
+//Searching for a song
+const songTitle = "3 nails in wood";
+let songByTitle = getSongByTitle(songDatabase, songTitle);
+console.log("Found song by title: ");
+console.log(songByTitle);
+
+//Create our own playlist
+let myPlaylist = [];
+let updatedMyPlaylist = addSongToMyPlaylist(songDatabase, myPlaylist,"yellow submarine");
+console.log("Added a new song to my the playlist: ");
+console.log(updatedMyPlaylist);
+
+function addSongToDatabase(songs) {  
+  const song = {
```

Comment:
This function will not be useful to anyone unless you want to add "Yellow Submarine" every time. I would put a `newSong` parameter as part of the function signature.